resources/views/ManageEvent/Orders.blade.php

```diff
@@ -114,12 +114,10 @@
                                 > {{$order->email}}</a>
                             </td>
                             <td>
-                                <a href="#" class="hint--top" data-hint="{{money($order->amount, $event->currency)}} + {{money($order->organiser_booking_fee, $event->currency)}} @lang("Order.organiser_booking_fees")">
-                                    {{money($order->amount + $order->organiser_booking_fee + $order->taxamt, $event->currency)}}
-                                    @if($order->is_refunded || $order->is_partially_refunded)
-
-                                    @endif
-                                </a>
+                                <span>{{ $order->getOrderAmount()->display() }}</span>
```

Comment:
The back-end orders dashboard the order total is displayed EX V.A.T is this intentional?

I think the getOrderAmount method needs to include the tax amount on the order.

Reply:
@samdb mentioned that all admin side amounts should exclude tax except the tax entries on the order details.